src/index.html

```diff
@@ -5,6 +5,7 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.3/semantic.min.css">
```

Comment:
This might look a bit stressful implementing this change but there's another `link` tag below loading the `semantic-ui` css. Looks like we have a duplicate, I suggest you remove one or any reason for doing it this way?